ctl/dump/dump.go

```diff
@@ -14,81 +14,97 @@
  * limitations under the License.
  */
 
-package dump
+ package dump
 
-import (
-	"fmt"
-	"io"
-	"net/http"
-	"os"
-
-	"github.com/spf13/cobra"
-
-	"kmesh.net/kmesh/ctl/utils"
-	"kmesh.net/kmesh/pkg/constants"
-	"kmesh.net/kmesh/pkg/logger"
-)
-
-const (
-	configDumpPrefix = "/debug/config_dump"
-)
-
-var log = logger.NewLoggerScope("kmeshctl/dump")
-
-func NewCmd() *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "dump",
-		Short: "Dump config of kernel-native or dual-engine mode",
-		Example: `# Kernel Native mode:
-kmeshctl dump <kmesh-daemon-pod> kernel-native
-	  
-# Dual Engine mode:
-kmeshctl dump <kmesh-daemon-pod> dual-engine`,
-		Args: cobra.ExactArgs(2),
-		Run: func(cmd *cobra.Command, args []string) {
-			_ = RunDump(cmd, args)
-		},
-	}
-	return cmd
-}
-
-func RunDump(cmd *cobra.Command, args []string) error {
-	podName := args[0]
-	mode := args[1]
-	if mode != constants.KernelNativeMode && mode != constants.DualEngineMode {
-		log.Errorf("Error: Argument must be 'kernel-native' or 'dual-engine'")
-		os.Exit(1)
-	}
-
-	cli, err := utils.CreateKubeClient()
-	if err != nil {
-		log.Errorf("failed to create cli client: %v", err)
-		os.Exit(1)
-	}
-
-	fw, err := utils.CreateKmeshPortForwarder(cli, podName)
-	if err != nil {
-		log.Errorf("failed to create port forwarder for Kmesh daemon pod %s: %v", podName, err)
-		os.Exit(1)
-	}
-	if err := fw.Start(); err != nil {
-		log.Errorf("failed to start port forwarder for Kmesh daemon pod %s: %v", podName, err)
-	}
-
-	url := fmt.Sprintf("http://%s%s/%s", fw.Address(), configDumpPrefix, mode)
-	resp, err := http.Get(url)
-	if err != nil {
-		log.Errorf("failed to make HTTP request: %v", err)
-		os.Exit(1)
-	}
-	defer resp.Body.Close()
-
-	body, err := io.ReadAll(resp.Body)
-	if err != nil {
-		log.Errorf("failed to read HTTP response body: %v", err)
-		os.Exit(1)
-	}
-
-	fmt.Println(string(body))
-	return nil
-}
+ import (
+	 "fmt"
+	 "io"
+	 "net/http"
+	 "os"
+ 
+	 "github.com/spf13/cobra"
+ 
+	 "kmesh.net/kmesh/ctl/utils"
+	 "kmesh.net/kmesh/pkg/constants"
+	 "kmesh.net/kmesh/pkg/logger"
+ )
+ 
+ const (
+	 configDumpPrefix = "/debug/config_dump"
+ )
+ 
+ var log = logger.NewLoggerScope("kmeshctl/dump")
+ 
+ func NewCmd() *cobra.Command {
+	 cmd := &cobra.Command{
+		 Use:   "dump",
+		 Short: "Dump config of kernel-native or dual-engine mode",
+		 Example: `# Kernel Native mode:
+ kmeshctl dump <kmesh-daemon-pod> kernel-native
+ 
+ # Dual Engine mode:
+ kmeshctl dump <kmesh-daemon-pod> dual-engine
```

Comment:
How about kmeshctl dump map? 

I am thinking we should not require users to specify dual-engine or kernel-native, we can get it automatically